update

```diff
@@ -61,37 +61,13 @@ def recalc_group_severity():
 def update_package_cache():
     print('    --> Querying alpm database...')
     packages = search('', filter_duplicate_packages=False, sort_results=False)
-    pkgbases = {}
-    packages_by_arch = defaultdict(set)
-    # assign packages per arch list (unify any into 64bit)
-    for package in packages:
-        arch = package.arch if package.arch != 'any' else primary_arch
-        packages_by_arch[arch].add(package.name)
-
-    print('    --> Collecting pkgbases...')
-    # query all pkgbases per arch
-    for arch in archs:
-        cmd = ['expac', '-t', '%s', '-S', '--config', get_configpath(arch), '%n %e %v %b']
-        cmd.extend(list(packages_by_arch[arch]))
-        bases = check_output(cmd).decode().split('\n')[:-1]
-        latest = None
-        for base in bases:
-            pkgname, pkgbase, version, builddate = (base.split(' '))
-            pkgbaseorname = pkgbase if pkgbase != '(null)' else pkgname
-            pkgbases[pkgname] = pkgbaseorname
-            if not latest or builddate > latest['builddate']:
-                latest = dict(pkg=pkgbaseorname, version=version, builddate=builddate)
-    latest_package = 'None' if not latest else \
-                     '{} {} {}'.format(latest['pkg'], latest['version'],
-                                       datetime.fromtimestamp(int(latest['builddate'])).strftime('%c'))
-    print('    --> Latest package: {}'.format(latest_package))
```

Comment:
The latest package functionality should be preserved to indicate how far the pulled database is behind